source/administration/bucket-replication.rst

```diff
@@ -244,8 +240,12 @@ workers operating on that queue. MinIO continuously works to replicate and
 remove objects from the queue while scanning for new unreplicated objects to
 add to the queue. 
 
-MinIO queues failed replication operations and retries those operations until replication succeeds.
-This helps keep replication up-to-date without relying on the MinIO scanner to notice unreplicated object versions.
+
+.. versionchanged:: RELEASE.2022-07-18T17-49-40Z
+
+   MinIO queues failed replication operations and retries those operations up to three (3) times.
+   
+   Replication operations that fail more than three times wait for the scanner to identify the object for replication before trying again.
```

Comment:
This _might_ put too much agency on the operation itself.

Maybe

```suggestion
MinIO dequeues operations which fail to replicate after three attempts.
The scanner can pick up those affected objects at a later time and requeue them for replication.
```

Reply:
👍